src/ucp/rndv/proto_rndv.c

```diff
@@ -112,6 +119,11 @@ static ucs_status_t ucp_proto_rndv_ctrl_select_remote_proto(
     rkey_config_key.sys_dev      = params->mem_info.sys_dev;
     rkey_config_key.mem_type     = params->mem_info.type;
     for (lane = 0; lane < ep_config->key.num_lanes; ++lane) {
+        if  (lane == ep_config->key.cm_lane) {
+            lanes_distance[lane] = ucp_cm_topo_default_distance;
```

Comment:
1. I prefer the previous version, instead of checking key.cm_lane explicitly in a protocols code
2. why not reuse ucs_topo_default_distance?
3. extra space after 'if' in line 122

Reply:
- CM and transport lanes are UCT/UCP terms, and not defined in UCS
 - common UCS topology logic is based on zero-default value by `+=`, so infinite start value wont work correctly

WDYT?

Reply:
but ucs_topo_default_distance defines 0 latency, inf bandwidth - which is the "neutral" value w.r.t composing distances.

Reply:
in case of CM lane I define it vice versa - inf lat and zero bandwidth since it cannot communicate and should not be selected for any protocol, is this wrong?

Reply:
I think ucs_topo_default_distance is the right choice when the distance is not relevant, for example here:

https://github.com/openucx/ucx/blob/6738305338039ce40a860bb2f6c43e3afdd3bc82/src/ucp/core/ucp_worker.c#L1931

we have other ways to prevent NULL_RESOURCE from being used for communications, here:

https://github.com/openucx/ucx/blob/6c6412abcd40ffea0708f6a7da3d83daa3a06e8d/src/ucp/proto/proto_common.c#L308